projects/canopy/src/lib/carousel/carousel.component.html

```diff
@@ -2,6 +2,31 @@
   <lg-heading [level]="headingLevel" class="lg-carousel__heading lg-visually-hidden">{{
     description
   }}</lg-heading>
+
+  <div
+    *ngIf="autoPlay"
+    aria-live="polite"
+    aria-atomic="true"
+    class="lg-carousel__autoplay"
+  >
+    <button
+      class="lg-carousel__autoplay__button lg-carousel__pause"
+      *ngIf="!pause.value"
+      (click)="pause.next(true)"
+    >
+      <span class="lg-visually-hidden">Stop Animation </span>
```

Comment:
Can we remove the space in both the visually hidden spans please? If you have added it for spacing you can use the `lgMargin` directive instead.

Reply:
I'll remove the space. Its not needed